docs/feature-gates.md

```diff
@@ -55,6 +56,17 @@ manifest provided as part of releases enables this feature by default. If you
 edit the manifest, make sure you do not disable it, as it is needed for correct
 NetworkPolicy implementation for Pod-to-Service traffic.
 
+### EndpointSlice
+
+`EndpointSlice` enable Service EndpointSlice support in AntreaProxy. The
+EndpointSlice API was introduced in Kubernetes 1.16 (alpha) and it is enabled
+by default in Kubernetes 1.17 (beta). This flag will take no effect if AntreaProxy
+is not enabled. EndpointSlice API version that AntreaProxy supports is v1beta1 for now,
+and other EndpointSlice API versions are not supported. The endpoint conditions 
+of `Serving` and `Terminating` are not supported currently. ServiceTopology will 
```

Comment:
"ServiceTopology is not supported either"?

Reply:
Agreed